templates/partials/search-results-candidate.html

```diff
@@ -26,7 +26,7 @@ <h3 class="result__title">
     <div class="result__column">
       {% if candidate.principal_committees %}
         {% with committee=candidate.principal_committees[0] %}
-        <span class="t-block t-bold">Principal Campaign Committee:</span>
+        <span class="t-block t-bold">Principal campaign committee:</span>
```

Comment:
Are you sure we're not capitalizing Principal campaign committees? Are they a proper noun?